chrome/fed-modules.json

```diff
@@ -258,6 +258,16 @@
                     },
                     "/ansible/inventory"
                 ]
+            },
+            {
+                "id": "groups",
+                "module": "./RootApp",
+                "routes": [
+                    {
+                        "pathname": "/insights/inventory/groups",
+                        "isFedramp": true
```

Comment:
how this boolean affects the nav entries? I assume it is related to one of the ongoing projects around fedramp.